### ISSUE TITLE
Cancel button for push/fetch

### ISSUE THREAD

**Issue by mtschoen** [ForkIssues/Tracker#254]
I connect to a few repositories over VPN, which means that if I don't have my VPN app running when I try to fetch/push in Fork, I have to wait for the operation to time out before I can try again. So it usually goes:

1. Try to fetch
2. "Oh yeah, open VPN"
3. VPN is connected
4. Wait 30s for Fork to timeout
5. Try again successfully.

I'd like to be able to hit a cancel button after step 2 :)

**Comment by DanPristupov** [comment]
Yes, absolutely. Pretty same thing happens when I press Fetch in some cafe with terrible connection.

This will be done soon, but I don't have an ETA yet.

**Comment by DanPristupov** [comment]
Hi,

I implemented ability to cancel long operations (fetch, pull, push, clone). It will be available publicly soon, but you can try it already in 1.0.67 pre-release (https://github.com/ForkIssues/Tracker/issues/287)
